docs/platform-services/threat-intelligence/find-threats.md

```diff
@@ -0,0 +1,26 @@
+---
+slug: /platform-services/threat-intelligence/find-threats
+title: Find Threats with Log Queries
+sidebar_label: Find Threats 
+description: Perform searches to find matches to data in threat intelligence indicators.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Once you [ingest threat intelligence indicators](/docs/platform-services/threat-intelligence/about-threat-intelligence/#ingest-threat-intelligence-indicators), you can perform searches to find matches to data in the indicators using the `threatlookup` search operator.
+
+The `threatlookup` operator allows you to search logs for matches in threat intelligence indicators. For example, use the following query to find logs in all `sec_record*` indexes with a `srcDevice_ip` attribute correlated to a threat indicator with a high confidence level (greater than 50): 
+
+ ```
```

Comment:
remove extra space in the beginning

Reply:
I'm not sure what you mean. When I view the article in my local build, I don't see any extra space.

Reply:
Weird! Mine is showing an extra space. I wasn't able to use the Suggest button because the backticks broke my suggestion. Here's a screenshot of my local build.
<img width="412" alt="Screenshot 2024-04-02 at 6 06 00 PM" src="https://github.com/SumoLogic/sumologic-documentation/assets/56411016/0212179f-0280-4173-804c-9a2c7e136c03">

Reply:
If you're talking about the space between the backticks and the text below it, that's normal. That's how it is for all my code examples.

Reply:
It's fixed now! See commit [Remove extra space](https://github.com/SumoLogic/sumologic-documentation/pull/3817/commits/b367ade9a82f2beb81ef78f407ea819e14ccfe39).